.github/workflows/wheels.yml

```diff
@@ -214,6 +199,50 @@ jobs:
           CIBW="build; args: --no-isolation"
           echo "CIBW_BUILD_FRONTEND=$CIBW" >> "$GITHUB_ENV"
 
+      - name: Windows - set environment variables
```

Comment:
I have a slight preference for keeping the common part unchanged, and adding the "ARM64 specific settings in a separate step. That shrinks the diff further, and also it's easier to see in the GitHub UI that the step didn't run for AMD64.

Reply:
Moved Arm specific to own step